vnext/Desktop.UnitTests/BaseWebSocketTests.cpp

```diff
@@ -25,12 +25,15 @@ using Error = IWebSocket::Error;
 // test macros.
 // clang-format off
 
-TEST_CLASS(BaseWebSocketTest){
+TEST_CLASS(BaseWebSocketTest)
+{
+  // Add any needed class-level attributes here.
+  // See https://docs.microsoft.com/en-us/visualstudio/test/microsoft-visualstudio-testtools-cppunittestframework-api-reference?view=vs-2019#test_class_attributes
   BEGIN_TEST_CLASS_ATTRIBUTE()
-  TEST_CLASS_ATTRIBUTE(L"Ignore", L"true")
   END_TEST_CLASS_ATTRIBUTE()
 
-  TEST_METHOD(CreateAndSetHandlers){
+  TEST_METHOD(CreateAndSetHandlers)
+  {
```

Comment:
Will Clang Format trample over this?

Reply:
No.
Some files, like MS unit tes files, have `clang-format` disabled because it messes up the structure horribly (likely due to the way they use macros).

Reply:
I agree. We've had issues where clang-format would produce incorrect formatting, especially in cases of lambdas or macros. If clang-format doesn't produce format that we need, we should either fix formatting rules in clang config files or turn clang-format off.

---
In reply to: [342888037](https://github.com/microsoft/react-native-windows/pull/3602#discussion_r342888037) [](ancestors = 342888037)